packages/css-library/src/stylesheets/formation-overrides/elements/typography.scss

```diff
@@ -22,6 +22,12 @@ body {
   font-size: 16px;
 }
 
+p {
+  line-height: vars.$base-line-height;
+  margin-bottom: 1em;
+  margin-top: 1em;
```

Comment:
Adding back the original paragraph styles because the v3 import has been removed.